code/game/gamemodes/wizard/artefact.dm

```diff
@@ -639,13 +639,16 @@ var/global/list/multiverse = list()
 	if(spooky_scaries.len >= 3 && !unlimited)
 		user << "<span class='warning'>This artifact can only affect three undead at a time!</span>"
 		return
-	M.makeSkeleton()
-	M.visible_message("<span class = 'warning'> A massive amount of flesh sloughs off [M] and a skeleton rises up!</span>")
-	M.revive()
+	if(heresy)
+		spawnheresy(M)//oh god why
```

Comment:
Does this actually bring them back from the dead? Or does it just desecrate corpses?